common/beerocks/bcl/include/bcl/son/son_wireless_utils.h

```diff
@@ -170,6 +170,7 @@ class wireless_utils {
     static uint8_t get_operating_class_by_channel(const beerocks::message::sWifiChannel &channel);
     static std::list<sChannelPreference>
     get_channel_preferences(const beerocks::message::sWifiChannel supported_channels[]);
+    static bool channel_validate(uint8_t operating_class, uint8_t channel);
```

Comment:
I think the naming should better indicate what the method is meant to do:
1. `channel_validate` can mean so many things - a `validate_channel_in_operating_class` would better explain itself
2. nitpick: it returns bool - so I prefer to have prefix `is_` to clarify we don't do an actual operation here - just perform a check. if you choose to accept this nitpick the naming would result in `is_channel_in_operating_class` - which I think is the best option in terms of self-explainable

Reply:
accepted!